tabbed-mode-explainer.md

```diff
@@ -20,22 +20,21 @@ Add a new manifest field `tab_strip` which allows apps to customize the tab stri
 
 ```
 "tab_strip": {
-    "home_tab": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "home_tab": "absent" | {
+        "icons": [...],
+        "scope_patterns": [{"pathname": "..."}]
     },
-    "new_tab_button": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "new_tab_button": "absent" | {
+        "url": <url>,
     },
 },
 ```
 
-The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should never navigate - links clicked from this tab should open in a new app tab. Apps can choose to customize the URL this tab is locked to and the icon displayed on the tab.
+The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should not navigate outside of the home tab scope (as specified by `scope_patterns`), so navigation outside of this scope should open in a new tab. If the `home_tab` field is unset it will default to `absent`. Apps can choose to customize the icon displayed on the tab.
 
-The new tab button, if present, should open a new tab at a URL that is within the scope of the app. The app may choose to set a custom URL and icon for this button.
+The `home_tab.scope_patterns` field allows the app to set a list of [URLPatterns](https://wicg.github.io/urlpattern/#urlpattern) to define the scope of the home tab. Navigation outide of this scope will be opened in a new tab, and navigation to a URL within the home tab scope will happen in the home tab. Only the pathname component of URLPattern will be used. If this field is unset, then the home tab scope will only include the `start_url`.
```

Comment:
I was confused by this and tried it:

```js
new URLPattern({pathname: '/foo'});
```

This sets everything but `URLPattern.pathname` to `"*"`. Should this instead initialize the `URLPattern.baseURL` to be the parsed `scope`?

![Screenshot 2023-02-15 at 11 41 21](https://user-images.githubusercontent.com/145676/219005131-41167e69-873c-4f87-8fb2-044d44b8937c.png)

Reply:
Yes, that's a good point - I'll add that in. Thanks!